pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/util/ValueReaderComparisons.java

```diff
@@ -0,0 +1,174 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.segment.local.io.util;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import org.apache.pinot.segment.spi.memory.PinotDataBuffer;
+
+public class ValueReaderComparisons {
+
+    private ValueReaderComparisons() {
```

Comment:
I just don't have my IDE set up like yours, and I tweaked the formatting manually until checkstyle passed. Has the project considered auto formatting with spotless as suggested [here](https://github.com/apache/pinot/pull/10000#issuecomment-1355938476)? I use it on other projects and it takes the pain out of maintaining consistent style without synchronising IDEs (not everyone uses IntelliJ) and IDE settings globally.

In the short term I will find the IDE settings and reformat the files I've changed here.

Reply:
Agreed. We have created #6790 to track this, but the work is not travail because we need to switch from intellij to eclipse format which is supported by spotless.